airbyte-integrations/bases/base-normalization/normalization/transform_catalog/utils.py

```diff
@@ -119,21 +107,22 @@ def is_big_integer(definition: dict) -> bool:
 
 
 def is_long(property_type, definition: dict) -> bool:
```

Comment:
can we remove the `property_type` param? Since it's redundant with the `definition` param

Reply:
@edgao this method may be called in different places. for some of them it's important to still have this `property_type`